src/main/java/com/github/f0xdx/Schemas.java

```diff
@@ -83,6 +87,12 @@ public static SchemaBuilder toBuilder(@NonNull Schema schema) {
     if (builder != null) {
       builder = builder.name(schema.name()).version(schema.version()).doc(schema.doc());
 
+      if (schema instanceof ConnectSchema) {
+        ConnectSchema connectSchema = (ConnectSchema) schema;
+        Optional<Map<String, String>> params = Optional.ofNullable(connectSchema.parameters());
+        builder = builder.parameters(params.orElse(Collections.emptyMap()));
```

Comment:
One minor nitpick: As Java immediately evaluates the expression in your `.orElse(...)` statement, an empty map is created each time the `connectSchema.parameters()` are extracted, whether those parameters exist or not. Therefore it's recommended to use the `.orElseGet(...)` variant for lazy evaluation.

```suggestion
        builder = builder.parameters(params.orElseGet(() -> Collections.emptyMap()));
```

Reply:
I didn't know that `orElse` was always evaluated. Thanks for the tip!